workflows/src/test/java/com/r3/corda/lib/tokens/workflows/NonFungibleTokenBuilderTests.java

```diff
@@ -0,0 +1,93 @@
+package com.r3.corda.lib.tokens.workflows;
+
+import com.r3.corda.lib.tokens.contracts.states.NonFungibleToken;
+import com.r3.corda.lib.tokens.contracts.types.IssuedTokenType;
+import com.r3.corda.lib.tokens.money.DigitalCurrency;
+import com.r3.corda.lib.tokens.money.FiatCurrency;
+import com.r3.corda.lib.tokens.workflows.utilities.NonFungibleTokenBuilder;
+import net.corda.core.identity.CordaX500Name;
+import net.corda.core.identity.Party;
+import org.junit.Test;
+
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+
+public class NonFungibleTokenBuilderTests {
+
+    @Test
+    public void NonFungibleTokenBuilderResolvesWithoutThrowing() throws TokenBuilderException, NoSuchAlgorithmException {
```

Comment:
method names should be starting with a lowercase.

Reply:
Adding fix then will merge!